dashboard/src/components/Accordion/Accordion.tsx

```diff
@@ -223,63 +228,51 @@ const AccordionTestsTrigger = ({
 const AccordionTestsContent = ({
   data,
 }: IAccordionTestContent): JSX.Element => {
-  const navigate = useNavigate({ from: '/tree/$treeId' });
-
-  const onClickRow = useCallback(
-    (testId: string) => {
-      navigate({
-        to: '/tree/$treeId/test/$testId',
-        params: {
-          testId: testId,
-        },
-        search: s => s,
-      });
-    },
-    [navigate],
-  );
-
   const rows = useMemo(() => {
-    return data.map(test => (
-      <TestTableRow key={test.id} test={test} onClick={onClickRow} />
-    ));
-  }, [data, onClickRow]);
+    return data.map(test => <TestTableRow key={test.id} test={test} />);
+  }, [data]);
 
   return (
     <div className="h-max-12 overflow-scroll">
-      <BaseTable headers={headerTestsDetails}>
-        <TableBody>{rows}</TableBody>
-      </BaseTable>
+      <BaseTable headers={headerTestsDetails}>{rows}</BaseTable>
     </div>
   );
 };
 
 interface ITestTableRow {
   test: TIndividualTest;
-  onClick: (testId: string) => void;
 }
 
-const TestTableRow = ({ test, onClick }: ITestTableRow): JSX.Element => {
-  const onClickHandle = useCallback(() => onClick(test.id), [onClick, test.id]);
+const TestTableRow = ({ test }: ITestTableRow): JSX.Element => {
+  const linkProps: LinkProps = {
+    to: '/tree/$treeId/test/$testId',
+    params: {
+      testId: test.id,
+    },
+    search: s => s,
+  };
```

Comment:
`useMemo`, objects will always create a new reference, `console.log({} === {})` is false